misc/safe-json/src/index.ts

```diff
@@ -1,14 +1,51 @@
+/** JSONStringify & JSONParse used from json-with-bigint */
```

Comment:
So this is extracted from https://github.com/sidorares/json-bigint I'm guessing right?

We should link back to it for reference and I think we technically have to include a link to the MIT license defined there.